proposals/simd/ImplementationStatus.md

```diff
@@ -145,6 +145,12 @@
 | `f64x2.convert_u/i64x2`    | `-munimplemented-simd128` |                    | :heavy_check_mark: | :heavy_check_mark: |
 | `v8x16.swizzle`            |                           |                    | :heavy_check_mark: |                    |
 | `v8x16.shuffle`            |                           |                    | :heavy_check_mark: | :heavy_check_mark: |
+| `i16x8.load8x8_u`          |                           |                    |                    |                    |
+| `i16x8.load8x8_s`          |                           |                    |                    |                    |
+| `i32x4.load16x4_u`         |                           |                    |                    |                    |
+| `i32x4.load16x4_s`         |                           |                    |                    |                    |
+| `i64x2.load32x2_u`         |                           |                    |                    |                    |
+| `i64x2.load32x2_s`         |                           |                    |                    |                    |
 | `i8x16.narrow_i16x8_s`     |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
 | `i8x16.narrow_i16x8_u`     |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
```

Comment:
I wonder if we should clear the checkmarks for all tools for renumbered instructions. Thoughts?

Reply:
Good question. Those used to work, maybe change them to question marks if GH markdown has those?

We should do that outside of this PR though, IMO :)  I'd be happy to help.

Reply:
I think it would make sense to do this change in the same PR since the old information becomes outdated the moment this gets merged, but I don't feel strongly about it. `:question:` (:question:) or `:grey_question:` (:grey_question:) would work well.

Reply:
Now I understand what you meant. That would work, or I can also move new instruction to the bottom of the list, that way we would not need to invalidate anything. I put extended loads next to load+splats, for consistency sake, but that can be debated.

Reply:
Moved new opcodes to the bottom of the list, now we would not need to invalidate renumbered instructions.